client/trino-client/src/test/java/io/trino/client/auth/external/TestExternalAuthenticator.java

```diff
@@ -169,10 +169,10 @@ public void testAuthenticationFromMultipleThreadsWithLocallyStoredToken()
                 .withResult(URI.create("http://token.uri"), successful(new Token("valid-token-4")));
```

Comment:
Please explain in commit message what is motivation behind this refactor. That this follows actual usage pattern of `ExternalAuthenticator` is used by concurrent jdbc connections.

Reply:
Typo in the commit message.